app/views/keywords/search.html.erb

```diff
@@ -0,0 +1,21 @@
+<div>
+  <div class="mb-2">
+    <p data-testid="search_result_search_term">Search term: <%= search_params[:search] %></p>
+    <p data-testid="search_result_query_type">Query type: <%= search_params[:query_type] %></p>
+  </div>
+
+  <p data-testid="search_result_total_matched_urls">Total matched URLs: <%= search_result_presenter.total_urls %></p>
```

Comment:
Let's use `I18n.t` for `Total matched URLs` and other place @Goose97

Reply:
Fixed in 4bdc770c852a0e732c6ea2f7ac897c4a36818a41